arkane/input.py

```diff
@@ -517,6 +517,7 @@ def bac(model_chemistry, bac_type='p', train_names='main',
         model_chemistry,
         bac_type=bac_type,
         db_names=train_names,
+        weighted=weighted,
```

Comment:
Since this is not in the docstring for `BACJob.__init__` as this gets passed to **kwargs we should make sure that this functionality is outlined in the documentation. If this is the case and I forgot to go back and delete this comment feel free to disregard this